### ISSUE TITLE
Need help on DB setup

### ISSUE THREAD

**Issue by ravinode** [aqnouch/Signal-Setup-Guide#23]
Hello I have successfully imported the project and created docker image on DB

can you help me on DB creation, I'm getting below error 

FATAL: database "signal_db" does not exist
	at org.postgresql.core.v3.ConnectionFactoryImpl.readStartupMessages(ConnectionFactoryImpl.java:684)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:199)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:66)
	at org.postgresql.jdbc2.AbstractJdbc2Connection.<init>(AbstractJdbc2Connection.java:127)
	at org.postgresql.jdbc3.AbstractJdbc3Connection.<init>(AbstractJdbc3Connection.java:29)
	at org.postgresql.jdbc3g.AbstractJdbc3gConnection.<init>(AbstractJdbc3gConnection.java:21)
	at org.postgresql.jdbc4.AbstractJdbc4Connection.<init>(AbstractJdbc4Connection.java:41)
	at org.postgresql.jdbc4.Jdbc4Connection.<init>(Jdbc4Connection.java:24)
	at org.postgresql.Driver.makeConnection(Driver.java:414)
	at org.postgresql.Driver.connect(Driver.java:282)
ERROR [2021-01-23 20:08:31,505] org.apache.tomcat.jdbc.pool.ConnectionPool: Unable to create initial connections of pool.

**Comment by aqnouch** [comment]
you must first to creat the databases in postgres, and populate them using this command:

`java -jar service/target/TextSecureServer-3.21.jar abusedb migrate service/config/config.yml
java -jar service/target/TextSecureServer-3.21.jar accountdb migrate service/config/config.yml
java -jar service/target/TextSecureServer-3.21.jar messagedb migrate service/config/config.yml`

before running the server

**Comment by ravinode** [comment]
Yeah I tried that, but later I found database name is given wrong in sample config vs docker file

**Comment by KashifAhmed** [comment]
I have changed the file name but nothing works
